manifests/09-prometheus-rules.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    name: image-registry-operator-alerts
+  name: image-registry-operator-alerts
+  namespace: openshift-image-registry
+spec:
+  groups:
+  - name: ImageRegistryOperator
```

Comment:
Same as here: https://github.com/openshift/cluster-samples-operator/pull/178/files#r328461466

Reply:
I sent a new commit that set these labels but before we move on: could you clarify the `info` severity level? I read on [1] that an info alert "never fires on its own" and I could not understand exactly when to use this severity then. You mean that an "info" severity rule has no expression or has an expression that never fires?


/cc @adambkaplan

[1]  https://github.com/openshift/cluster-samples-operator/pull/178/files#r328461466

Reply:
Sorry, yes I should have been more clear. Info alerts are no different in their definition other than the `severity: info` label, they just never cause a notification on their own (due to the way that Alertmanager is configured), only if in combination with a warning or critical alert in the same group.

Reply:
The idea is that they are largely informational, and don't necessarily have meaning on their own, but could be a first hint to investigate if something is truly on fire.

Reply:
OK, I get it. Thanks for the explanation.

Reply:
@adambkaplan 

I have set the following severities:

1. ImageRegistryMissingStorageConfiguration as **WARNING**
2. ImageRegistryDegraded as **CRITICAL**
3. ImageRegistryStorageBackendReconfigured as **WARNING**

Do you think these "values" are adequate?

Reply:
I'd say ImageRegistryStorageBackendReconfigured should have `severity: info`. As an administrator I don't want to receive notifications about actions that I've just made.

Reply:
@dmage Migrated it to `info`.

Reply:
Resolving this one as the changes were already implemented.